packages/react-bootstrap-table2-example/stories/index.js

```diff
@@ -156,20 +162,20 @@ import RemoteCellEdit from 'examples/remote/remote-celledit';
 import RemoteAll from 'examples/remote/remote-all';
 
 // css style
-import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Hi @AllenFang, 
Instead of importing `style` from `node_module`, I decided to serve the style as static files alternatively. The `pros` is that we could dynamically import style in our stories; the `cons` is that we have to store the files in the code base and ask for the style every time as long as we switch to another story. I'm not sure it's good or not. Please let me know if you have any concern, thanks :)